internal/operator-controller/applier/helm_test.go

```diff
@@ -295,20 +322,61 @@ func TestApply_InstallationWithPreflightPermissionsEnabled(t *testing.T) {
 		require.Nil(t, objs)
 	})
 
-	t.Run("fails during installation", func(t *testing.T) {
+	t.Run("fails during installation because of missing RBAC rules", func(t *testing.T) {
```

Comment:
This isn't the correct test description right? PreAuthorize is returning `nil` for missing rules and an error string, which mocks out what happens when pre-auth fails, not when pre-auth returns missing rules.

Reply:
Yep, I think so, accepting the suggestion.

Reply:
Looks like I actually just duplicated this test case, the next one is the same and has the proper description. I'll remove this extra one for now and we will add the other suggested test cases in a follow-up issue.